packages/lit-dev-tools-cjs/src/api-docs/generate.ts

```diff
@@ -11,14 +11,13 @@ import {execFile} from 'child_process';
 import {promisify} from 'util';
 import {ApiDocsTransformer} from './transformer.js';
 import {lit2Config} from './configs/lit-2.js';
-import {litElement2Config} from './configs/lit-element-2.js';
-import {litHtml1Config} from './configs/lit-html-1.js';
 
 import type {ApiDocsConfig} from './types.js';
 
 const execFileAsync = promisify(execFile);
 
-const configs = [lit2Config, litElement2Config, litHtml1Config];
+// Only generate documentation for most recent Lit versions.
+const configs = [lit2Config];
```

Comment:
If we're dropping `litElement2Config` and `litHtml1Config`, should there be some artifact that we check in that gets used for future site builds? Maybe we also want to keep around the config srcs in case we do need to regenerate those at some point?

Reply:
Aha, right! So then my remaining question is whether we might want to keep around the config files without deleting in case we ever need to regenerate them. Though if we start upgrading the generator, maybe the configs might not be compatible?

I guess the point of archiving is that we won't be updating these docs anyway.

Reply:
I do think it'd be nice to have some way of reconstructing the config that generated those JSON files. Maybe it's as simple as a README in the folder that points to this PR and/or instructions on which commit to check out and which commands to run.

Reply:
Good idea. I can add a section to the README.